app/views/devise/registrations/new.html.erb

```diff
@@ -36,7 +36,11 @@
         <label  class="checkboxterm"><a class="linkterminos" href="">He leído y acepto los términos y condiciones</a></label></br>
     </input> -->
   </div>
-  <%= recaptcha_tags %>
+  <% if @show_checkbox_recaptcha %>
+    <%= recaptcha_tags(site_key: Rails.application.secrets.reCaptcha_key) %>
+  <% else %>
+    <%= recaptcha_v3(action: 'user_registration', site_key: Rails.application.secrets.reCaptcha_key) %>
+  <% end %>  
```

Comment:
According to the gem documentation, the tags are configured with a v2 recaptcha: https://github.com/ambethia/recaptcha?tab=readme-ov-file#examples

This would men we need both keys

Reply:
It seems to me that this will never show the challenge